tests/Http/Three/SignupTest.php

```diff
@@ -143,110 +143,157 @@ public function testSignupsIndex()
     }
 
     /**
-     * Test for signup index with included post info.
+     * Test for retrieving all signups as admin.
+     * Admins should see why_participated in response.
      *
-     * GET /api/v3/signups?include=posts
+     * GET /api/v3/signups
      * @return void
      */
-    public function testSignupIndexWithIncludedPosts()
+    public function testSignupsIndexAsAdmin()
     {
-        $signup = factory(Signup::class)->create();
-        $posts = factory(Post::class, 5)->create();
-
-        foreach ($posts as $post) {
-            $post->signup()->associate($signup);
-            $post->status = 'accepted';
-            $post->save();
-        }
+        factory(Signup::class, 10)->create();
 
-        $response = $this->getJson('api/v3/signups' . '?include=posts');
+        $response = $this->withAdminAccessToken()->getJson('api/v3/signups');
 
         $response->assertStatus(200);
-
         $response->assertJsonStructure([
             'data' => [
                 '*' => [
-                    'posts' => [
-                        'data' => [
-                            '*' => [
-                                'id',
-                            ],
-                        ],
-                    ],
+                    'id',
+                    'northstar_id',
+                    'campaign_id',
+                    'campaign_run_id',
+                    'quantity',
+                    'source',
+                    'details',
+                    'created_at',
+                    'updated_at',
+                    'why_participated',
+                ],
+            ],
+            'meta' => [
+                'cursor' => [
+                    'current',
+                    'prev',
+                    'next',
+                    'count',
                 ],
             ],
         ]);
     }
 
     /**
-     * Test for signup index with included rejected post info. as admin and non-admin/non-owner.
+     * Test for retrieving all signups as owner.
+     * Signup owner should see why_participated in response.
      *
-     * GET /api/v3/signups?include=posts
+     * GET /api/v3/signups
      * @return void
      */
-    public function testSignupIndexWithIncludedPostsWithMultipleCredentials()
+    public function testSignupsIndexAsOwner()
     {
-        $post = factory(Post::class)->create();
-        $signup = $post->signup;
+        $signups = factory(Signup::class, 10)->create();
 
-        // Test with annoymous user that no posts are returned.
-        $response = $this->getJson('api/v3/signups' . '?include=posts');
+        foreach ($signups as $signup) {
+            $signup->northstar_id = $this->faker->northstar_id;
+            $signup->save();
+        }
+
+        $response = $this->withAccessToken($signup->northstar_id)->getJson('api/v3/signups');
```

Comment:
I think this test might be failing since `$signup` is scoped to the for loop above?